regen-network-1/genesis.tmpl.json

```
@@ -0,0 +1,185 @@
+{
+  "genesis_time": "2021-04-08T16:00:00Z",
+  "chain_id": "regen-network-1",
```

I thoguht we landed on `regen-1` for chain-id ?
```suggestion
  "chain_id": "regen-1",
```

---

@glandua ?

---

I remember landing on regen-1 as well, however I did like the naming convention of regen-ledger-1 (2,3,4 etc) as it differentiates the blockchain from the token ($regen) and various entities (Regen Foundation, Regen Development) etc.  From the perspective of interchain naming etc, regen-1 is clearly better.  Perhaps We can still refer to it as regen ledger in repos, marketing etc, but the name of regen-1 is affixed to the chain.